### PR TITLE
Will tam contrib

### PR DESCRIPTION
**Pull request by will-tam**

### CODE REVIEW

**Review by alevoski — APPROVED** [review]
ok + resolve with open() issue

File: CS/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import sys
+
+sys.path.insert(0, 'CS')
```
Comment (alevoski) [inline CS/__init__.py]: ok with : sys.path.insert(0, sys.path[0]+'/CS')

**Review by alevoski — APPROVED** [review]
ok !

**Review by alevoski — APPROVED** [review]
ok !

**Review by alevoski — APPROVED** [review]

**Review by alevoski — APPROVED** [review]

File: CS/__init__.py (hunk 1 above)
Comment (will-tam) [reply CS/__init__.py]: Done !

**Review by alevoski — APPROVED** [review]
ok !

**Pull request merged**